bin/run-browser-tests-ci-batch

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python3
+
+# DOC: Run a batch of browser tests in CI mode. Takes one argument, the
+# DOC: 1-indexed batch number.
+
+import glob
+import math
+import subprocess
+import sys
+
+BATCH_COUNT = 4
+
+def bad_batch_number_input(batch_num_input):
+    print(f"Invalid batch number: {batch_num_input}", file=sys.stderr)
+    exit(1)
+
+if len(sys.argv) < 2:
+    bad_batch_number_input("none")
+
+try:
+    batch_number = int(sys.argv[1])
+except ValueError:
+    bad_batch_number_input(sys.argv[1])
+
+if batch_number < 1 or batch_number > BATCH_COUNT:
+    bad_batch_number_input(batch_number)
+
+test_files = glob.glob('browser-test/src/*.test.ts')
+test_files = [file.replace("browser-test/", "") for file in test_files]
+test_files.sort()
+test_files_count = len(glob.glob('browser-test/src/*.test.ts'))
+batch_size = math.ceil(test_files_count / BATCH_COUNT)
+
+batch_contents = test_files[(batch_number - 1) * batch_size : batch_number * batch_size]
```

Comment:
If (batch_number * batch_size) is greater than the list size, does this error?

Reply:
nope, python will silently take the minimum of the provided value or the length of the list

Reply:
```python
>>> [1, 2, 3][1:2000]
[2, 3]
```

Reply:
Cool, thanks!